consensus-types/blocks/setters.go

```diff
@@ -179,6 +180,9 @@ func (b *SignedBeaconBlock) SetExecutionRequests(req *enginev1.ExecutionRequests
 	if b.version < version.Electra {
 		return consensus_types.ErrNotSupported("SetExecutionRequests", b.version)
 	}
+	if req == nil {
+		return errors.New("nil execution request")
```

Comment:
should I throw an error or just set it to an empty request object?

Reply:
In my opinion it's bad practice to have different ways of handling the same thing within one package. We don't check this in other setters, so we should just skip it here too.